packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -14,7 +14,7 @@ import {
     SelectNative,
     Text,
 } from '@deriv/components';
-import { routes, platforms, WS, shouldHideOccupationField } from '@deriv/shared';
+import { routes, platforms, WS, shouldHideOccupationField, isTablet } from '@deriv/shared';
```

Comment:
why do we need `isTablet` from `@deriv/shared`

Reply:
Ahhh. It was added automatically. Let me remove this. VS code added this 😬